hpcbench/driver.py

```diff
@@ -989,14 +1000,17 @@ def _jinja_executor_template(self):
 
     def _write_executor_script(self, ostr):
         """Write shell script in charge of executing the command"""
+        environment = self.execution.get('environment') or {}
```

Comment:
certainly looks cleaner, but I think the above change breaks the logic and we cannot control the override on the level of single modules or environment variables